operator/cmd/mesh/upgrade.go

```diff
@@ -267,6 +275,21 @@ func checkSupportedVersions(cur, tar, versionsURI string) error {
 	return nil
 }
 
+func generateDefaultTelemetryManifest(l *Logger) (map[string]*object.K8sObject, error) {
```

Comment:
guess we can move this func into upgrade_hooks.go, or create a new source file for telemetry hook to avoid passing the telemetry map[string]*object.K8sObject.